### ISSUE TITLE
Easing Styles

### ISSUE THREAD

**Issue by ghost** [ROBLOX/Studio-Tools#59]
I can't edit easing styles and the editor just deletes poses when i try to open the easing editor

**Comment by Speedist** [comment]
Yeah, It kind of works when it wants to for me. like 90% of the time it deletes the pose, and 10% of the time it prompts. maybe adding a prompt option when you click on the pink boxes. so it the options say "delete" and the other "easing" or maybe just have something appear on a spot of the pink box that changes into a different color entirely  to signify to select it in easing mode
EDIT: I figured out how to get the easing menu open. just hold ctrl then click the pink box which the key frame for the limb is. _then_ click it and it will bring up the menu
EDIT2: Nevermind, It was just the gui of the easing part glitching out, it seems like it the button for the easing editor doesn't come out at all I feel like the function is covered by the delete function
